homeassistant/components/iotty/switch.py

```diff
@@ -0,0 +1,99 @@
+"""Implement a iotty Light Switch Device."""
+import logging
+from typing import Any
+
+from iottycloud.lightswitch import LightSwitch
+from iottycloud.verbs import LS_DEVICE_TYPE_UID
+
+from homeassistant.components.switch import SwitchDeviceClass, SwitchEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .api import IottyProxy
+from .const import DOMAIN
+from .coordinator import IottyDataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class IottyLightSwitch(SwitchEntity):
```

Comment:
```suggestion
class IottyLightSwitch(SwitchEntity, CoordinatorEntity[Device]):
```

Please adjust the entity